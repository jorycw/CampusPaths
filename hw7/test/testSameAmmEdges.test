# Find a path with Dijkstra's algorithm when there are two distinct paths
# in which both contain two edges, but one has a lower cost

CreateGraph graph
AddNode graph a
AddNode graph b
AddNode graph c
AddNode graph d

AddEdge graph a b .5
AddEdge graph b d 3.6

AddEdge graph a c 2
AddEdge graph c d 2

FindPath graph a d