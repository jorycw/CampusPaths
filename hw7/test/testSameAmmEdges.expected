# Find a path with Dijkstra's algorithm when there are two distinct paths
# in which both contain two edges, but one has a lower cost

created graph graph
added node a to graph
added node b to graph
added node c to graph
added node d to graph

added edge 0.500 from a to b in graph
added edge 3.600 from b to d in graph

added edge 2.000 from a to c in graph
added edge 2.000 from c to d in graph

path from a to d:
a to c with weight 2.000
c to d with weight 2.000
total cost: 4.000